devices/niko.js

```diff
@@ -198,27 +232,31 @@ module.exports = [
         model: '552-721X1',
         vendor: 'Niko',
         description: 'Single connectable switch',
-        fromZigbee: [fz.on_off, local.fz.switch_operation_mode, local.fz.switch_action],
-        toZigbee: [tz.on_off, local.tz.switch_operation_mode],
+        fromZigbee: [fz.on_off, local.fz.switch_operation_mode, local.fz.switch_action, local.fz.switch_status_led],
+        toZigbee: [tz.on_off, local.tz.switch_operation_mode, local.tz.switch_led_enable, local.tz.switch_led_state],
         configure: async (device, coordinatorEndpoint, logger) => {
             const endpoint = device.getEndpoint(1);
             await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff']);
             await reporting.onOff(endpoint);
             await endpoint.read('manuSpecificNiko1', ['switchOperationMode']);
+            await endpoint.read('manuSpecificNiko1', ['outletLedState']);
+            await endpoint.read('manuSpecificNiko1', ['outletLedColor']);
         },
         exposes: [
             e.switch(),
             e.action(['single', 'hold', 'release']),
             exposes.enum('operation_mode', ea.ALL, ['control_relay', 'decoupled']),
+            exposes.binary('led_enable', ea.ALL, true, false).withDescription('Enable LED'),
+            exposes.binary('led_state', ea.ALL, true, false).withDescription('LED State'),
```

Comment:
Instead of a boolean, `ON`/`OFF` fits better here.

Reply:
led_enable is already a boolean on the outlets, but swapping led_State to ON/OFF might make sense given most 'state' properties use ON/OFF (or LOCKED/UNLOCKED, ...)

Reply:
I’d personally leave led_enable as a boolean as we have other _enable’s that are boolean.

So only led_state should be ON/OFF I think, although consistency on this generally not great across z2m.

Reply:
Makes sense. I personally also think a boolean for led_enable makes more sense. I will wait for @Koenkk to weigh in before reverting the change to led_enable. There do seem to be quite a few different approaches across z2m to handle this.